spec/models/claim/advocate_claim_spec.rb

```diff
@@ -23,31 +23,47 @@
     let(:same_provider_external_user) { create(:external_user, provider: provider) }
     let(:other_provider_external_user) { create(:external_user, provider: other_provider) }
 
-    it 'raises error message if no external user is specified' do
-      subject.external_user_id = nil
-      expect(subject).not_to be_valid
-      expect(subject.errors[:external_user_id]).to eq(['Choose an advocate'])
+    context 'with no external user' do
+      before do
+        subject.external_user_id = nil
```

Comment:
**nit/extension-of-refactor**: as a refactor I would also use named subject, but i see this does not define one explictly